screener-e2e/examples/test/configs/wdio.screener.local.conf.js

```diff
@@ -0,0 +1,276 @@
+exports.config = {
+    //
+    // ====================
+    // Runner Configuration
+    // ====================
+    //
+    // WebdriverIO allows it to run your tests in arbitrary locations (e.g. locally or
```

Comment:
Please remove all comments and not used config code and just add 1 comment at the top that more info can be found here https://webdriver.io/docs/configurationfile.html. This makes the config easier to read and only focus on the real changes for Screener to be used

Reply:
Done, fyi, I got it from your wdio examples :)